src/services/sheet.service.ts

```diff
@@ -0,0 +1,32 @@
+import { fetchJson } from '../utils/fetch.ts';
+
+interface SheetsResponse {
+  type: string;
+  data: [];
+  offset: number;
+  total: number;
+}
+
+export type Sitemap = SiteMapEntry[];
+
+export type SiteMapEntry = {
```

Comment:
Hier auch - das hat Samu in einen shared-type ausgelagert & umbenannt

Reply:
s.o.